SPECS/moby-engine/moby-engine.spec

```diff
@@ -3,8 +3,8 @@
 
```

Comment:
I believe you will also need to change the commit_hash to 87a90dc786bda134c9eb02adbae2c6a7342fb7f6

Reply:
yes this need to be updated

Reply:
Could you also verify that spec still uses the correct version of libnetwork
see line #3139 -> Source1: https://github.com/moby/libnetwork/archive/master.tar.gz/#/%{upstream_name}-libnetwork-%{version}.tar.gz

Reply:
done, thank you!